contracts/cw1/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "cw1"
+version = "0.1.2"
+authors = ["Jan Woźniak <jan@confio.gmbh>"]
+edition = "2021"
+description = "todo"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/sylvia"
+homepage = "https://cosmwasm.com"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
```

Comment:
This one is not a complete contract, it doesn't need to be `cdylib`. `cdylib` is only for WASM binaries, so final contracts, anything else is just rlib.